source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -284,7 +291,23 @@ Response:
 <backup id="backup-uuid">
     <from_checkpoint_id>previous-checkpoint-uuid</from_checkpoint_id>
     <disks>
-        <disk id="disk-uuid" />
+        <disk id="disk-uuid" backup_mode="incremental" />
+        ...
+        ...
+    </disks>
+    <phase>initiailizing</phase>
+    <creation_date>
+</backup>
+```
+
+Response for an incremental backup which includes disks
+that wasn't part of the given `<from_checkpoint_uuid>`:
+```
```

Comment:
please use ```xml

Reply:
I will add the tag in a different patch for the entire file.